lib/src/segment_config.dart

```diff
@@ -1,22 +1,25 @@
+enum SegmentIntegrationItemType { amplitude, unknown }
+
 class SegmentConfig {
   SegmentConfig({
     required this.writeKey,
     this.trackApplicationLifecycleEvents = false,
-    this.amplitudeIntegrationEnabled = false,
     this.debug = false,
+    this.integrationItems,
   });
 
   final String writeKey;
   final bool trackApplicationLifecycleEvents;
-  final bool amplitudeIntegrationEnabled;
+  final List<SegmentIntegrationItemType>? integrationItems;
   final bool debug;
 
   Map<String, dynamic> toMap() {
     return {
       'writeKey': writeKey,
       'trackApplicationLifecycleEvents': trackApplicationLifecycleEvents,
-      'amplitudeIntegrationEnabled': amplitudeIntegrationEnabled,
       'debug': debug,
+      'integrationItems':
+          integrationItems?.map((e) => e.toString().split(".").last).toList(),
```

Comment:
whats the idea with this `split`?

Reply:
From enumeration `SegmentIntegrationItemType. amplitude` and so on. We just need to get `amplitude`. That's how we get it.

Reply:
@cdmunoz 
Sorry, I'm not using 2.15 yet, and I thought it would be more compatible with my implementation